### DISCUSSION TITLE
Acess mode 'roc' can miss a latch

### DISCUSSION THREAD

**Discussion by arnfol** [esynr3z/corsair#130] [Questions]
It is connected to issue #28. 

While for `rolh` and `roll` access modes not-latching event was fixed in #37, this problem still exists for `roc`. If `roll/rolh` could be compared to their latched value, we cannot do the same for `roc`. 
So, the question is:
@esynr3z should `roc` always be accompanied by enable (`ie` hardware options)? I have not found any rules on this in the documentation, but I think it has little sense without both enable and input.

**Comment by esynr3z** [comment]
Good question. I'm restoring the context at the moment. I'll get back to this soon.
